### PR TITLE
Max chars input validation

### PR DESCRIPTION
**Pull request by autogun**
Adding `max_chars` config parameter to a text field.

I sometime require limit user input to specific number of chars. I've been doing this until now with-in the executed script but I find it much useful this way.

### CODE REVIEW

File: src/model/parameter_config.py [hunk 1]
```diff
@@ -277,6 +279,9 @@ def validate_value(self, value, *, ignore_required=False):
             return None
 
         if self.type == 'text':
+            if (not is_empty(self.max_chars)) and (len(value) > int(self.max_chars)):
+                return 'is greater than allowed char length (' \
```
Comment (bugy) [inline src/model/parameter_config.py:283]: Could you change "greater" to "longer?

File: src/model/parameter_config.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
     'type',
     'min',
     'max',
+    'max_chars',
```
Comment (bugy) [inline src/model/parameter_config.py:30]: may be we could call it "max_length"? What do you think?

**Comment by autogun** [comment]
Got a little confused and mistakenly pushed something else. Please have another look. Apologies for that.

**Comment by autogun** [comment]
@bugy is there anything else required here?

**Comment by bugy** [comment]
Oh, damn, i forgot about it, sorry :(

**Review by bugy — APPROVED** [review]

**Pull request merged**